clients/src/main/resources/common/message/FetchSnapshotResponse.json

```diff
@@ -17,7 +17,7 @@
   "apiKey": 59,
   "type": "response",
   "name": "FetchSnapshotResponse",
-  "validVersions": "0",
+  "validVersions": "0-1",
```

Comment:
Please add a comment to mention what changed in v.1.

Reply:
Fixed.